### ISSUE TITLE
[ANDROID] Home feed not implemented

### ISSUE THREAD

**Issue by patricknasc** [firebase/friendlypix#61]
The home feed in android's version isn't implemented yet.

**Comment by Divala** [comment]
Yes I just noticed the same

**Comment by migueltarga** [comment]
What is it supposed to do?

**Comment by patricknasc** [comment]
I analyzed the code and my best attempt to solve this issue was doing the following change to PostFragment.java: http://pastebin.com/9yNYBDUL

Unfortunately I can't consider this issue solved and I'm not sure if this could be considered a solution. For me it shouldn't be, but I couldn't find a better approach.  

What I did was: 

1) I moved the declaration of postPaths to outside 
`
FirebaseUtil.getFeedRef().child(FirebaseUtil.getCurrentUserId())
                        .addValueEventListener...
`
This resulted in line 190:  
`final List<String> postPaths = new ArrayList<>();`

2) I moved the initialization of mAdapter to outside onDataChange. For some reason I couldn't understand, it wasn't being called never. So that, no post couldn't be rendered on Home feed.
` mAdapter = new FirebasePostQueryAdapter(postPaths,
                                new FirebasePostQueryAdapter.OnSetupViewListener() {
                                    @Override
                                    public void onSetupView(PostViewHolder holder, Post post, int position, String postKey) {
                                        Log.d(TAG, "onSetupView - post: " + post.toString());
                                        Log.d(TAG, "onSetupView - position: " + position);
                                        setupPost(holder, post, position, postKey);
                                    }
                                });`
3) I had to force at least one postPath hardcoded (line 191) to allow the rendering of the others posts that are already in the Home feed. That's the worst thing about this solution in my opinion. It's odd, but if I comment this line, no post will be displayed, but if I let it, all of the posts will be displayed correctly. This could be accepted as a "work around" if every new user receives a default welcome post (with a fixed id), so that all of the followed user post's will be displayed correctly. 

4) I had to replace the line                
```
FirebaseUtil.getFeedRef().child(FirebaseUtil.getCurrentUserId())
                        .addListenerForSingleValueEvent

``` 
by
```
 FirebaseUtil.getFeedRef().child(FirebaseUtil.getCurrentUserId())
                        .addValueEventListener
```
so that all of the posts saved in current user's feed could be rendered. Otherwise, only the one post would. 

For the sake of readiness, I'll share my complete version of PostFragment.java here: http://pastebin.com/YVZnt6Jp (I just have to change the hardcoded post id (line 191) by one you already have in your posts list (for instance, a welcome post)

The current version of this file in repo can be found here: http://pastebin.com/zEY9ESYr or here: https://github.com/firebase/friendlypix/blob/master/android/app/src/main/java/com/google/firebase/samples/apps/friendlypix/PostsFragment.java

I would really appreciate if somebody else could give a try in this issue as well! I unfortunately couldn't succeed exactly as I expected.
